test/e2e/upgrade_test.go

```diff
@@ -240,3 +252,32 @@ func (tc *testContext) testWorkloadsAvailable(t *testing.T) {
 	})
 	assert.NoError(t, err)
 }
+
+// waitForCSINodes waits for a CSINode to exist for each node with a driver loaded, indicating CSI functionality has
+// been enabled for the given node.
+func (tc *testContext) waitForCSINodesWithDrivers(nodes []v1.Node) error {
+	return wait.PollImmediate(retry.Interval, 10*time.Minute, func() (bool, error) {
```

Comment:
Lets stop introducing a deprecated function.

Reply:
Leaving this as is so it can be backported to 4.13